manifest.json

```diff
@@ -108,6 +108,15 @@
       "includes_header": true,
       "format": "hostname"
     },
+    {
+      "file": "ms_phishing_simulation_domains.txt",
+      "identifier": "ms_phishing_simulation_domains",
```

Comment:
seems a little too specialized for lists? I don't think we have other ones in here

Reply:
agree, there's no use for this i can think of outside of the global exclusion. in which case, we should prob hard code it in the Exclusion rule

Reply:
@aidenmitchell can you turn this into a standalone rule that we can add as a Global Exclusion instead?

Reply:
@aidenmitchell can you turn this into a standalone rule that we can add as a Global Exclusion instead?

(fysa: @morriscode )